exercises/dnd-character/example.exs

```diff
@@ -0,0 +1,29 @@
+defmodule DndCharacter do
+  defstruct [
+    :strength,
+    :dexterity,
+    :constitution,
+    :intelligence,
+    :wisdom,
+    :charisma,
+    :hitpoints
+  ]
+
+  def modifier(score), do: :erlang.floor((score - 10) / 2)
+
+  def ability(), do: :rand.uniform(16) + 2
+
+  def character() do
+    constitution = ability()
+
+    %__MODULE__{
+      strength: ability(),
+      dexterity: ability(),
+      constitution: constitution,
+      intelligence: ability(),
+      wisdom: ability(),
+      charisma: ability(),
+      hitpoints: 10 + modifier(constitution)
+    }
+  end
+end
```

Comment:
It's not strictly necessary but it would be nice to have the example file be consistent with stub file (for example, having the same typespecs). In other words if you use the stub as the starting point for this file, then fill in the implementation, it gives some nice symmetry.